src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.html

```diff
@@ -1 +1,2 @@
-<dot-edit-layout-grid [gridContainers]="dummyContainers"></dot-edit-layout-grid>
+<button (click)="saveLayout()">Save</button>
+<dot-edit-layout-grid #editLayoutGrid [pageView]="pageView"></dot-edit-layout-grid>
```

Comment:
why this component receive all the PageView object? I think it should receive only the Layout object

Reply:
@hmoreras Also use async pipe here